tuned/plugins/plugin_cpu.py

```diff
@@ -133,7 +133,7 @@ def _instance_init(self, instance):
 			try:
 				self._cpu_latency_fd = os.open(consts.PATH_CPU_DMA_LATENCY, os.O_WRONLY)
 			except OSError:
-				log.error("Unable to open '%s', disabling PM_QoS control" % consts.PATH_CPU_DMA_LATENCY)
+				log.info("Unable to open '%s', disabling PM_QoS control" % consts.PATH_CPU_DMA_LATENCY)
```

Comment:
If the file does not exist, we should consider PM_QoS control unsupported, so we should log it as a warning. If the file could not be opened for other reasons, error level seems appropriate. So you could add an if-else statement here and change the log level as appropriate. I don't think hiding the message with an info level makes sense.

A related change we might want to do here is open the file descriptor only when it's needed - I believe there are tunings supported by the plugin that don't require the file descriptor.